src/components/elements/LoadingSkeleton.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Skeleton } from '@mui/material';
+import { BoxProps } from '@mui/system';
+import times from 'lodash-es/times';
+import React from 'react';
+
+interface Props extends BoxProps {
+  animation?: 'pulse' | 'wave' | false;
+  count?: number;
+  height?: number;
+}
+const LoadingSkeleton: React.FC<Props> = ({
+  animation = 'wave',
+  count = 6,
+  height = 50,
+  ...props
+}) => {
+  return (
+    <Box aria-live='polite' aria-busy='true' {...props}>
+      {times(count, (i) => (
+        <Skeleton
+          animation={animation}
+          key={i}
+          height={height}
+          variant='rounded'
+          sx={{ mb: 2 }}
+        />
```

Comment:
Can we do:
```
Array.from({ length: count }, (x, i) => (
        <Skeleton
          animation={animation}
          key={i}
          height={height}
          variant='rounded'
          sx={{ mb: 2 }}
        />
));
```

The test is complaining about the lodash import, but maybe its just the style. Trying `import { times } from 'lodash-es';` should also work, pretty sure it tree shakes to only whats imported.